migtests/scripts/run-validate-bulk-assessment-report.sh

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env bash
+
+# The script takes up to two arguments:
+# 1. TEST_NAME: The name of the test to run (mandatory).
+# 2. env.sh: An optional environment setup script to source before running the test. If not provided, this file will be picked from the ${TESTS_DIR}
+
+set -e
+
+if [ $# -gt 2 ]
+then
+	echo "Usage: $0 TEST_NAME [env.sh]"
+	exit 1
+fi
+
+set -x
+
+export YB_VOYAGER_SEND_DIAGNOSTICS=false
+export TEST_NAME=$1
+
+export REPO_ROOT="${PWD}"
+export SCRIPTS="${REPO_ROOT}/migtests/scripts"
+export TESTS_DIR="${REPO_ROOT}/migtests/tests"
+export TEST_DIR="${TESTS_DIR}/${TEST_NAME}"
+
+export PYTHONPATH="${REPO_ROOT}/migtests/lib"
+
+# Order of env.sh import matters.
+if [ $2 != "" ] #if env.sh is passed as an argument, source it
+then
+    if [ ! -f "${TEST_DIR}/$2" ]
+	then
+		echo "$2 file not found in the test directory"
+		exit 1
+	fi
+	source ${TEST_DIR}/$2
+else
+	source ${TEST_DIR}/env.sh
+fi
+source ${SCRIPTS}/${SOURCE_DB_TYPE}/env.sh
+
+source ${SCRIPTS}/functions.sh
+
+export SOURCE_DB_TYPE="oracle"
+export BULK_ASSESSMENT_DIR=${BULK_ASSESSMENT_DIR:-"${TEST_DIR}/bulk-assessment-dir"}
+export SOURCE_DB_SCHEMA=${SOURCE_DB_SCHEMA:-"TEST_SCHEMA"}
+export SOURCE_DB_SCHEMA2=${SOURCE_DB_SCHEMA2:-"TEST_SCHEMA2"}
+export EXPORT_DIR1=${EXPORT_DIR1:-"${BULK_ASSESSMENT_DIR}/ORCLPDB1-TEST_SCHEMA-export-dir"}
```

Comment:
you can use the above variables here:
export EXPORT_DIR1=${EXPORT_DIR1:-"${BULK_ASSESSMENT_DIR}/${SOURCE_DB_NAME}-${SOURCE_DB_SCHEMA}-export-dir"}

that way if the other variable is changes, this will automatically get updated.

"Change at once place in code, reflected at all"